app/Libraries/Controller.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace App\Libraries;
+
+class Controller
+{
+  /**
+   * Load models
+   *
+   * @param string $model
+   * @return object
+   */
+  public function model($model): object

```

Comment:
See if you can use dependency injection / the Reflection API to replace the usage of this method completely.